build.sbt

```diff
@@ -16,21 +16,7 @@ val jsSettings = Def.settings(
 )
 
 val nativeSettings = Def.settings(
-  libraryDependencies += "org.scala-native" %%% "junit-runtime" % nativeVersion % Test,
-  addCompilerPlugin("org.scala-native" % "junit-plugin" % nativeVersion cross CrossVersion.full),
-  pomPostProcess := { node =>
-    import scala.xml._
-    import scala.xml.transform._
-    new RuleTransformer(new RewriteRule{
-      override def transform(n: Node) =
-        if (n.label == "dependency" && (n \ "artifactId").text.startsWith("junit-runtime_native"))
-          NodeSeq.Empty
-        else
-          n
-    }).transform(node)(0)
```

Comment:
We can revert https://github.com/typelevel/shapeless-3/pull/65 because it was fixed upstream.